crawl-ref/source/dat/des/arrival/small.des

```diff
@@ -3412,3 +3412,126 @@ MAP
   ...........
    .........
 ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_battle_scene
```

Comment:
Fitting with the undocumented guideline about escapes: Add a few more doors, a hatch, or even both

This vault has good flavour. If you're willing to use this as a learning project, figuring out how to place some random blood spatter would be a nice addition to the vault.

Reply:
I added that to the guidelines file, in a separate commit.